tests/Meilisearch.Tests/DocumentTests.cs

```diff
@@ -516,19 +516,19 @@ public async Task GetMultipleExistingDocuments()
         {
             var index = await _fixture.SetUpBasicIndex("GetMultipleExistingDocumentTest");
             var documents = await index.GetDocumentsAsync<Movie>();
-            Assert.Equal(7, documents.Count());
-            documents.First().Id.Should().Be("10");
-            documents.Last().Id.Should().Be("16");
+            Assert.Equal(7, documents.Results.Count());
+            documents.Results.First().Id.Should().Be("10");
+            documents.Results.Last().Id.Should().Be("16");
         }
 
         [Fact]
         public async Task GetMultipleExistingDocumentsWithLimit()
         {
             var index = await _fixture.SetUpBasicIndex("GetMultipleExistingDocumentWithLimitTest");
             var documents = await index.GetDocumentsAsync<Movie>(new DocumentQuery() { Limit = 2 });
-            Assert.Equal(2, documents.Count());
-            documents.First().Id.Should().Be("10");
-            documents.Last().Id.Should().Be("11");
+            Assert.Equal(2, documents.Results.Count());
+            documents.Results.First().Id.Should().Be("10");
+            documents.Results.Last().Id.Should().Be("11");
```

Comment:
I miss two tests:

- one for `GetDocumentAsync` with `fields`
- one for `GetDocumentsAsync` with `fields` in the `DocumentsQuery`